### PR TITLE
Update Delete command Tests for Edge Cases and Validations

### LINKED ISSUE #96: Add delete command tests
Currently, the Delete command is operational with our latest implementation of the Add feature; therefore, only additional tests have been introduced to cover edge cases.

### PR DESCRIPTION
**Pull request by rizkidelta**
Expanded test coverage to include boundary conditions and invalid scenarios. Closes #96.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 1]
```diff
@@ -82,6 +82,37 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
...
+
+    @Test
+    public void execute_deleteInEmptyList_throwsCommandException() {
+        Model emptyModel = new ModelManager(); // Assuming this creates an empty address book
+
+        Index someIndex = Index.fromOneBased(1);
+        DeleteCommand deleteCommand = new DeleteCommand(someIndex);
+
+        assertCommandFailure(deleteCommand, emptyModel, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+    }
```
Comment (yisiox) [inline src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:115]: Should we add a newline here?

**Review by yisiox — COMMENTED** [review]
Just a nit

**Review by yisiox — APPROVED** [review]
LGTM
